src/core/logging.mjs

```diff
@@ -77,7 +77,7 @@ export const Logger = class {
         // - Write all logs with importance level of `info` or less to `solo.log`
         //
         new winston.transports.File({ filename: `${constants.SOLO_LOGS_DIR}/solo.log` })
-        // new winston.transports.File({filename: constants.TMP_DIR + "/logs/error.log", level: 'error'}),
+        // new winston.transports.File({filename: constants.TMP_DIR + "/output/error.log", level: 'error'}),
```

Comment:
this commented out line can probably be deleted, else you should change it back to /logs/ since it refers to a solo directory on the local machine